### PR TITLE
Edit user 136180469

### PR DESCRIPTION
**Pull request by rodrigoulisses**

### CODE REVIEW

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 
     <%= f.input :role, as: :string, wrapper_html: {class: 'col-xs-12 u-mt-medium'}, label: false do %>
-      <%= f.input_field :role, as: :select, collection: roles, include_blank: t('simple_form.placeholders.user.role'), class: 'input' %>
+      <%= f.input_field :role, as: :select, collection: roles, include_blank: t('simple_form.placeholders.user.role'), class: 'input', disabled: cannot?(:manager, User) %>
       <span class="helper" aria-describedby="">Admin: pode convidar outras pessoas | Editor: visualiza e edita mapas</span>
     <% end %>
 
     <div class="col-xs-12 u-mt-medium">
-      <%= f.submit 'Cadastrar', class: 'button button-primary u-fill' %>
+      <%= f.submit t('users.buttons.save'), class: 'button button-primary u-fill' if can? :manager, User %>
```
Comment (dmitryrck) [inline app/views/users/_form.html.erb]: :ok_hand:

File: spec/feature/authorize_spec.rb [hunk 2]
```diff
@@ -114,10 +124,20 @@
...
+      expect(page).to have_field 'Nome do colaborador', disabled: true
+      expect(page).to have_field 'Email do colaborador', disabled: true
+      expect(page).to have_select 'user_store_id', disabled: true
+      expect(page).to have_select 'user_role', disabled: true
+      expect(page).to_not have_button 'Salvar'
 
       visit '/users/new'
 
       expect(page).to have_content 'Oppsssss... Você não tem permissão para realizar esta edição.'
+
```
Comment (dmitryrck) [inline spec/feature/authorize_spec.rb]: Pra que essa linha? :thinking:

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -1,28 +1,28 @@
 <%= simple_form_for @user, html: { role: 'form', wrapper: false } do |f| %>
   <div class="row">
     <%= f.input :name, wrapper_html: {class: 'col-xs-12 u-mt-medium'}, label: false do %>
-      <%= f.input_field :name, class: 'input' %>
+      <%= f.input_field :name, class: 'input', disabled: (@user.persisted? or cannot? :manager, User) %>
```
Comment (marclerodrigues) [inline app/views/users/_form.html.erb]: Would be better to create a helper with this logic.

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -23,6 +27,16 @@ def create
     end
   end
 
+  def update
```
Comment (marclerodrigues) [inline app/controllers/users_controller.rb]: Why don't you test these controller actions?

**Pull request merged**